lib/proto/build.rs

```diff
@@ -0,0 +1,29 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let workspace_dir = env!("CARGO_WORKSPACE_DIR");
+    let proto_include_dir = format!("{workspace_dir}proto");
+    let proto_tuxtape_dir = format!("{proto_include_dir}/tuxtape");
```

Comment:
Can we use`Path`s for things that are paths?

```suggestion
    let workspace_dir = Path::new(env!("CARGO_WORKSPACE_DIR"));
    let proto_include_dir = workspace_dir.join("proto");
    let proto_tuxtape_dir = workspace_dir.join("tuxtape");
```